operator/internal/securedcluster/values/translation/translation.go

```diff
@@ -334,6 +334,37 @@ func (t Translator) getCollectorContainerValues(collectorContainerSpec *platform
 	}
 
 	cv.AddChild(translation.ResourcesKey, translation.GetResources(collectorContainerSpec.Resources))
+	runtimeConfig := collectorContainerSpec.RuntimeConfig
+	if runtimeConfig != nil {
+		cv.AddChild("runtimeConfig", t.getRuntimeConfig(runtimeConfig))
+	}
```

Comment:
`getRuntimeConfig` is nil-safe so just:

```suggestion
	cv.AddChild("runtimeConfig", t.getRuntimeConfig(collectorContainerSpec.RuntimeConfig))
```